1.7/administration/installing/high-availability-setup.md

```diff
@@ -0,0 +1,89 @@
+---
+post_title: Hybrid Cloud and Multi-Datacenter
+menu_order: 6
+---
+
+DC/OS supports multi-availability zone (AZ) configurations. This topic describes the setup recommendations and caveats.
+
+### Terminology
```

Comment:
I think we need one more fault domain, which could be rack (on-prem) or placement group (AWS). It should be noted, however, that placement groups on AWS don't actually provide any guarantee that they're in a different fault domain.